doc/ruleSets/PER.rst

```diff
@@ -0,0 +1,10 @@
+=================
+Rule set ``@PER``
```

Comment:
Evolving recommendations have names instead of numbers. Title of this document is “PER Coding Style”, but I don't expect there to be any other PERs that we could have a rule set for. Also, `PERCS` doesn't read well :)

Reply:
> I don't expect there to be any other PERs that we could have a rule set for

Just showing `PER` is fine for this reason IMO.